src/face_detector/launch/face_detector.launch

```diff
@@ -1,15 +1,18 @@
 <launch>
-	<node name = "camera" pkg = "usb_cam" type = "usb_cam_node">
-		<param name = "video_device" value = "/dev/video0" />
+	<node name="camera" pkg="usb_cam" type="usb_cam_node">
+		<param name="video_device"   value="/dev/video0" />
+		<param name="io_method"      value="mmap" />
+		<param name="pixel_format"   value="yuyv" />
+		<param name="image_witdh"    value="640" />
+		<param name="image_height"   value="480" />
```

Comment:
width is misspelled. Though I think those dimensions are the defaults anyway, so you can likely leave them off.